templates/store/publisher-details.html

```diff
@@ -0,0 +1,212 @@
+{% extends webapp_config['LAYOUT'] %}
+
+{% block meta_copydoc %}{% endblock %}
+
+{# TODO: title, meta, etc #}
+{% block meta_title %}{{ publisher }} in the Snap Store{% endblock %}
+{% block meta_description %}Install {{ publisher }} software for Linux in the Snap Store{% endblock %}
+
+{% if icon_url %}
+  {% block meta_image %}
+    {{ icon_url }}
+  {% endblock %}
+{% endif %}
+
+
+{% block content %}
+  <div class="p-strip--dark is-shallow" style="background: #262626">
```

Comment:
If we don't use` p-strip--dark` anywhere else can we override the default vanilla `#111` with `#262626`?

Reply:
👍